obspy/core/util/base.py

```diff
@@ -153,6 +153,7 @@ def create_empty_data_chunk(delta, dtype, fill_value=None):
         dtype = native_str(dtype)
     if fill_value is None:
         temp = np.ma.masked_all(delta, dtype=np.dtype(dtype))
+        temp.data[:] = np.nan
```

Comment:
Just put zeros there. I mean it really only for tests and you want to avoid any C routines using the masked data which might contain anything. zeros should be pretty save. `np.ma.zeros` also exists.